docs/getting-started.md

```diff
@@ -193,77 +192,82 @@ data will be lost. Keep in mind, that it may take a much longer time from 8V to
 
 {% include end-panel.html %}
 
-{% include end-section.html %}
+<div class="row">
+<div class="col-md-8" markdown="1">
+### Shutting down
+
+You'll often want to turn off your EV3 while it isn't in use. When you are ready
+to wish your EV3 good night, turn if off by pressing the *back* button from the
+main menu in brickman or pressing _and holding_ the *back* button from any screen
+in brickman. This will open a dialog where you can select *Power Off* to turn
+off the EV3. _If you're using the command line, running `sudo poweroff` in a
+terminal works too._
+</div>
```

Comment:
`*word*` and `_word_` have the same effect in markdown - *italic*. If you intended **bold**, you need `**word**`, otherwise, let's be consistent and just use `*` everywhere.